parsec/core/gui/tr/parsec_en.po

```diff
@@ -2053,6 +2053,17 @@ msgstr "No limit to the number of users"
 msgid "TEXT_ORG_INFO_USER_LIMIT-limit"
 msgstr "Limited to {limit} active users"
 
+msgid "TEXT_ORG_INFO_SEQUESTRATION_TOOLTIP"
+msgstr ""
+"The sequestration allows an administrator to gain access to every data "
```

Comment:
see the helper text in bootstrap cli: https://github.com/Scille/parsec-cloud/pull/2510/files#diff-f77e61f4e11da30ca7289047bbdb9dbadaff1c54f608022489056ca872ee80c6R19-R33

```A sequestered organization is able to ask it users to encrypt
their data with third party asymetric keys (called "sequester service").

Those "sequester services" must themselve be signed by the "sequester authority" key
that is configured during organization bootstrap.
This have the following implications:
- All data remains encrypted
- The sequester services are in position to bypass end-to-end user encryption
- The Parsec Server has no way of modifying the sequester services keys
- Sequester services can be added and removed to handle key rotation
- A regular organization cannot be turned in a sequestered one (and vice-versa)

A typicall usecase for sequestered organization is data recovery or legal requirement for
company to have access to all data produced by former employees.
```